osrm/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM osrm/osrm-backend:latest
 
-RUN apk add python3
+RUN apk add --no-cache python3 && \
```

Comment:
yikes. all of these lines are necessary? what changed?

Reply:
Copied shamelessly from https://github.com/frol/docker-alpine-python3/blob/master/Dockerfile

Reply:
should we put that as a source to explain? what changed that broke it?